### ISSUE TITLE
Create some realistic example pages with cookie banner and page to present at the accessibility clinic

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk-frontend#2097]
## What
We're going to [take the cookie banner to the accessibility clinic](https://github.com/alphagov/design-system-team-internal/issues/386) for a review. 

We should be able to amend the prototype we used for the accessibility research to include the new cookie banner and page and deploy it to a new herokup app.

## Why
We need some realistic examples of pages and a service flow to take to the accessibility clinic. 

## Who needs to know about this
Designers, developers

## Done when 
- [ ] We've identified how we're going to include the cookie banner and page in the prototype 
- [ ] We've added the cookie banner and page in the prototype
- [ ] We've deployed the prototype to a new heroku app
